modules/govuk/manifests/apps/email_alert_api/checks.pp

```diff
@@ -33,4 +33,24 @@
     from      => '1hour',
     desc      => 'email-alert-api - high number of delivery attempts have not received status updates',
   }
+
+  @@icinga::check::graphite { 'email-alert-api-warning-subscription-contents':
+    ensure    => $ensure,
+    host_name => $::fqdn,
+    target    => 'transformNull((keepLastValue(stats.gauges.govuk.email-alert-api.subscription_contents.warning_total))',
+    warning   => '1',
+    from      => '1hour',
+    desc      => 'email-alert-api - unprocessed subscription contents created over 10 minutes ago (or 35 minutes ago if during publishing times)',
```

Comment:
I can see the benefit of including these numbers in the description, but I would be worried that they get out of sync with the numbers in email-alert-api. Perhaps we could add a comment to the code in email-alert-api to remind people to update the alert here?

Reply:
Thanks @thomasleese ! I did debate whether to keep this in, I've taken it out for now and add a comment as you have suggested. I'll also include it in the docs for this alert.